osh/hub/osh_xmlrpc/worker.py

```diff
@@ -27,6 +27,9 @@
 from osh.hub.service.csmock_parser import unpack_and_return_api
 from osh.hub.waiving.results_loader import TaskResultsProcessor
 
+if settings.ENABLE_FEDORA_MESSAGING:
+    from fedora_messaging import api, message
```

Comment:
This dependency is not covered by the packaging.  We should mention this requirement in `settings.py`.

Reply:
> This dependency is not covered by the packaging. We should mention this requirement in `settings.py`.

Where exactly this dependency should be specified in `settings.py`?

Reply:
I would mention it in the comment so that one knows what needs to be installed before enabling the option.

Reply:
I added it as a comment above `ENABLE_FEDORA_MESSAGING = False` line in `settings.py`.